yarnitor/background/worker.py

```diff
@@ -296,10 +308,9 @@ def run_task(app):
             # Due to timing things happening here some of our calls may fail
             except requests.exceptions.ReadTimeout:
                 pass
-            except Exception as e:
-                import traceback
-                logging.error("Error for application %s %s", app["id"], app["name"])
-                logging.error(traceback.format_exc())
+            except Exception as ex:
+                logger.error("Error for application %s %s", app["id"], app["name"])
+                logger.exception(ex)
```

Comment:
`logger.exception` should render the same info without the explicit imports / long-hand code.